chef-utils/lib/chef-utils/dist.rb

```diff
@@ -87,6 +87,11 @@ class Solo
       EXEC = "chef-solo"
     end
 
+    class Workstation
+      # The suffix for Chef Workstion's /opt/chef-workstation or C:\\opscode\chef-workstation
```

Comment:
you had a typo here in the comment that was failing the spellcheck test

Reply:
oops!